docker-compose.yml

```diff
@@ -8,6 +8,10 @@ x-test-variables: &test-variables
     POSTGRES_USER: 'bob'
     POSTGRES_DB: 'db'
 services:
+    jaeger:
+        image: jaegertracing/all-in-one
+        ports:
+            - "127.0.0.1:16686:16686"
```

Comment:
Does this mean that Jaeger will come up automatically in the dev and test environments? In the dev environment, would we want to drop a note in the README about how they'd visit the UI? 

And in the test environment, is it actually possible to test this in some way? I'm thinking it would probably be super difficult. They have [an API](https://www.jaegertracing.io/docs/1.18/apis/) but it would be very awkward to check it after sending stuff. Let me know what you think.

Reply:
It does come up by default, but I actually wasn't sure if we even wanted to add this to the docker-compose? It sounds like it may be a good idea, especially that we are adding server-side tracing. If we want to keep it, then yeah we can add a note to the README.

For testing, I keep thinking about separation of concerns: we integrate with Opentelemetry, which then integrates with Jaeger. So, we could just test our Opentelemetry part, which if works, implies Jaeger will work too.